sdk/src/connections/target/addi9036_sensor.cpp

```diff
@@ -349,9 +349,11 @@ Addi9036Sensor::setFrameType(const aditof::FrameDetails &details) {
     /* Set the frame format in the driver */
     CLEAR(fmt);
     fmt.type = m_implData->videoBuffersType;
-#if defined FXTOF1
+
+    if (TARGET == "toybrick") {
```

Comment:
This will make compiler throw a warning "comparison with string literal results in unspecified behavior". To compare two C strings in C or C++ use strncmp().

Reply:
Also, while working on something else I noticed that this is not the right approach. The sdk/CMakeLists.txt no longer defines TARGET to "toybrick" but instead adds a new define: TOYBRICK. So you should use that.